package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "big.js": "^6.1.1",
```

Comment:
* I get this error when i run `npm start` kindly fix this error as it prevents the app from starting 
![image](https://user-images.githubusercontent.com/52497246/143464077-754b8397-848d-4e18-b25b-ccde998bbac2.png)

Reply:
@Davidosky007 everything works well on my end, I tried to clone and start app several times, how can I check the error?

Reply:
* After `npm install` i run `npm start` then i get this error

Reply:
Dear future reviewer, If you have the same issue, please run `npm audit fix --force`, because even if I run it on my end issues are not fixed, I asked my friend to try to `npm start` after force fix everything worked.
I would be very thankful if you contacted me on slack before asking to re-review. Thank you.

Reply:
It works on my system without an issue ✔️

Reply:
@sinansevgi thank you